src/styles/blocks/stop-watch.scss

```diff
@@ -0,0 +1,64 @@
+.stop-watch {
+  position: absolute;
```

Comment:
It's not necessary  to use here position: absolute. It would be better if you use display:flex instead.